package.json

```diff
@@ -14,89 +14,55 @@
     "build": "tsc && tsc -p tsconfig.esm.json",
     "build:docs": "npx typedoc",
     "eslint": "eslint --ext .ts src test",
-    "jest": "jest --coverage",
+    "test": "jest --coverage",
     "lint": "npm run prettier && npm run eslint",
     "prettier": "prettier --check \"src/**/*.ts\" \"test/**/*.ts\"",
     "prettier:write": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "prepublishOnly": "npm run build",
-    "test": "npm run lint && npm run jest",
     "test:nonetwork": "npm run lint && npm run jest -- --testPathIgnorePatterns network.spec"
   },
-  "files": [
-    "dist/",
-    "README.md"
-  ],
+  "files": ["dist/", "README.md"],
   "engines": {
-    "node": ">=14"
+    "node": ">=22.0.0"
   },
   "dependencies": {
     "debug": "^4.1.1",
     "env-paths": "^2.2.0",
-    "fs-extra": "^8.1.0",
     "got": "^11.8.5",
     "progress": "^2.0.3",
     "semver": "^6.2.0",
     "sumchecker": "^3.0.1"
```

Comment:
Do we want to bump major versions here while we're at it?

Reply:
Makes sense to do so to the best of our abilities. I'll give it a shot!